modules/support-generating-a-sosreport-archive.adoc

```diff
@@ -31,7 +31,24 @@ $ oc get nodes
 ----
 $ oc debug node/my-cluster-node
 ----
-
++
+To enter into a debug session on the target node that is tainted with the `NoExecute` configuration, add a toleration to a dummy namespace, and then have the debug pod created in the dummy namespace:
```

Comment:
Consider an alternative:

"If the target node is tainted with the `NoExecute` effect, add a toleration to a dummy namespace, and start the debug pod in the dummy namespace:"

* While "configuration" seems OK, the KCS and the doc on creating the taint mention "effect".
* "then have the debug pod created..." is unnecessary passive voice.  Please revise in any way that uses active voice--you aren't obliged to accept my sugg.

Reply:
@mikemckiernan Thanks for the suggestion. When we say "start the debug pod", it would mean that the user has an action item to do - starting the debug pod, which is not the case. The debug pod gets started automatically. How about changing it to this: "add a toleration to a dummy namespace so that the debug pod starts in the dummy namespace"?

Reply:
My understanding is that when our customer enters the `oc debug node/<node_name>` command, that action by our customers starts the debug pod.  If it is not the customer who starts the pod by running the command, do you have a contact who can clarify what starts the debug pod so that the sentence can be rewritten with active voice?